Connection.php

```diff
@@ -271,6 +282,11 @@ public function open()
         if ($this->_socket !== false) {
             return;
         }
+        // twemproxy/nutcrackes doesn't support DB selection
+        if($this->twemproxy) {
+            // in this case database must be always 0.
+            $this->database = 0;
```

Comment:
if only the database needs to be zero for twemproxy, I do not think the connection class needs to be aware of this.